ntlm3/HTTPNtlmAuthHandler.py

```diff
@@ -17,6 +17,7 @@
 import socket

```

Comment:
Is this file even needed in the main section, shouldn't it be in test for our testing and people use requests-ntlm instead when wanting to use python-ntlm3 in a http sense?